### DISCUSSION TITLE
install tailwindcss

### DISCUSSION THREAD

**Discussion by meredwan** [meXuvo/reactjs_starter#3] [Q&A] [answered]
how to install tailwindcss with this skeleton

**Comment by risuvoo** [comment, marked as answer]
> how to install tailwindcss with this skeleton

Go to tailwindcss documentation [tailwindcss](https://tailwindcss.com/docs/guides/create-react-app)

**Reply by meredwan** [reply]
Thanks

**Reply by risuvoo** [reply]
Welcome
